BiasAndFairnessServers/src/crud/bias_and_fairness.py

```
@@ -0,0 +1,43 @@
+from database.db import get_db
+from sqlalchemy import text
+
+async def get_metrics_by_id(id, db):
```

Pylint (3.3.7)

[warning] Redefining built-in 'id'

(W0622)